### ISSUE TITLE
Feature request : Implement "Ask" option in download directory setting, so you are asked which folder to put the download into

### ISSUE THREAD

**Issue by capzrfun** [localsend/localsend#1281]
### Is your feature request related to a problem? Please describe.

Well its just a little annoying if you have your download directory set, and you wanna save to another folder you cant just quickly accept the transfer you must go into options then change the directory for it to download to. 

### Describe the solution you'd like

Implement "Ask" option in download directory setting, so you are asked which folder to put the download into when the transfer initially comes. This in addition to the current setting which allows you to select a default download folder

**Comment by Tienisto** [comment]
When someone sends you a file, you can tap on "Options" which is right above Decline and Accept

**Comment by black-ish** [comment]
> When someone sends you a file, you can tap on "Options" which is right above Decline and Accept

True, but it has to be done everytime you receive a file, having a default "ask" option would be great.

**Comment by MrCatEsq** [comment]
@Tienisto, thank you for your work. Is there a plan to add this option? I also really miss having a "select folder manually every time" checkbox in the settings, or at least an "accept as" button (similar to "Save As").
